JcMinarro/Philology
# Override all methods in philology resources

Because we attach new wrapped context to support translations, the other resources (dimensions, colors) that we're trying to get from the Resources rely on initial Resources and in result we get non-correct values if `android:configChanges="orientation"` attribute is applied for the Activity in `Manifest.xml`. Also, the `onConfigurationChanged` method isn't called for any custom views.

### Changes
 1. Override all remaining methods in `PhilologyResources` and used the `baseResources` field to get all values except text resources.
2. Create `CustomView` and update the sample project to test configuration changes.

### How to test
1. Run the sample project.
2. Observe text is changing on text view at the bottom on orientation change.
